C-Plus-Plus/cp/replace_pi.cpp

```diff
@@ -0,0 +1,63 @@
+/* Problem statement: 
+You are given a string, you have to replace all the occurrences of "pi" int the string into 3.14.
+Example :
```

Comment:
Great work! A very small change needs to be made. I/O should be at last, after the code.